keyless/pepper/service/src/lib.rs

```diff
@@ -94,11 +112,20 @@ fn process_common(
     exp_date_secs: u64,
     epk_blinder: Vec<u8>,
     uid_key: Option<String>,
+    derivation_path: Option<String>,
     encrypts_pepper: bool,
     aud: Option<String>,
-) -> Result<Vec<u8>, ProcessingFailure> {
+) -> Result<(Vec<u8>, Vec<u8>, AccountAddress), ProcessingFailure> {
     let config = Configuration::new_for_devnet();
 
+    let derivation_path = if let Some(path) = derivation_path {
```

Comment:
nit: `unwrap_or_else`. the linter probably complaining about it?

Reply:
it's now *shrug*